pymedphys/labs/pedromartinez/graticule/qc-graticule.py

```diff
@@ -68,6 +69,8 @@
 
 import utils as u
 
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "utils"))
```

Comment:
No need to adjust the path

Reply:
That was my firts attempt at relative modules. :) I'm trying to follow your example above in _mudensity but no success so far

Reply:
One option is to import like:

```python
import pymedphys.labs.pedromartinez.utils as utils
```

which whould work if utils.py exists at the path `pymedphys/labs/pedromartinez/utils`.

A neater option though could be to split out your CLI tool into a separate package and make it callable from the pymedphys CLI in the same way as the following tools:

https://github.com/pymedphys/pymedphys/tree/master/pymedphys/cli